.github/workflows/lint_python.yml

```diff
@@ -7,10 +7,11 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
       - run: pip install --upgrade pip wheel
-      - run: pip install bandit codespell darker flake8 flake8-2020 flake8-bugbear
+      - run: pip install bandit black codespell darker flake8 flake8-2020 flake8-bugbear
                          flake8-comprehensions isort mypy pytest pyupgrade safety
       - run: bandit --recursive --skip B101 . || true  # B101 is assert statements
-      - run: darker --check .
+      - run: black --check .
```

Comment:
We intentionally have some non-Black formatting in the code base. This serves as dogfooding so when we run Darker on its own code base, it will skip reformatting those regions unless we've made modifications to them.

So it has been a conscious choice to *not* run Black on Darker's code base.